scripts/src/plh-data-convert/index.ts

```diff
@@ -157,6 +157,20 @@ function convertXLSXSheetsToJson(xlsxFilePath: string) {
   const workbook = xlsx.readFile(xlsxFilePath);
   const { Sheets } = workbook;
   Object.entries(Sheets).forEach(([sheet_name, worksheet]) => {
+    /* If bold or italics, include HTML in cell value */
```

Comment:
This code should all be in it's own function, with a comment to explain better why we need it